charts/tempo-distributed/values.yaml

```diff
@@ -1523,11 +1523,11 @@ storage:
     # -- The supported storage backends are gcs, s3 and azure, as specified in https://grafana.com/docs/enterprise-traces/latest/configure/reference/#admin_client_config
     backend: filesystem
 
-# Global overrides
+# -- The standard overrides configuration section. This can include a `defaults` object for applying to all tenants (not to be confused with the `global` property of the same name, which overrides `max_byte_per_trace` for all tenants). For an example on how to enable the metrics generator using the `global_overrides` object, see the 'Activate metrics generator' section below. Refer to [Standard overrides](https://grafana.com/docs/tempo/latest/configuration/#standard-overrides) for more details.
```

Comment:
> This can include a `defaults` object for applying to all tenants (not to be confused with the `global` property of the same name, which overrides `max_byte_per_trace` for all tenants).

Ok, I'm confused by this. Is it just pointing out that it's weird we have a "global_overrides.default.global" setting? The second global meaning "related to lots of different components" in tempo?

Reply:
The `global_overrides` section is just used wholesale for the `overrides` block: https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/values.yaml#L1420

Unfortunately it's the naming (see first comment) that's calling people out. I'd agree the entire overrides objects in the chart need work and re-templating, but this PR was intended just to try and remove confusion from current settings.